cogs/logging.py

```diff
@@ -14,6 +14,55 @@ def __init__(self, bot):
         self.bot = bot
         self.logger = bot.logger
 
+    @commands.group(aliases=['ldbc', 'get_these_errors_outta_here'])
+    @checks.is_owner()
```

Comment:
This should be @commands.is_owner since it is coming from discord.py not my checks file